lib/runners/processor/flake8.rb

```diff
@@ -48,12 +50,15 @@ def setup
     end
 
     def analyze(changes)
+      parallel = [true, nil].include?(config_linter[:parallel])
+
       capture3!(
         analyzer_bin,
         "--exit-zero",
         "--output-file", report_file,
         "--format", OUTPUT_FORMAT,
         "--append-config", IGNORED_CONFIG_PATH,
+        "--jobs", parallel ? "2" : "1",
```

Comment:
@ybiquitous 
```suggestion
        "--jobs", parallel ? "auto" : "1",
```
The default value for Flake8 `jobs` option is `auto`; Flake8 determine the optimal parallelism with the option. It is better that we set `auto` when `parallel` option is `true` in my opinion.
https://flake8.pycqa.org/en/latest/manpage.html#options

Reply:
Make sense. 👍 b4a67019